onebusaway-android/build.gradle

```diff
@@ -148,4 +151,8 @@ dependencies {
     compile 'org.onebusaway.util:comparators:1.0.0'
     // Only used on Amazon devices via amazon build flavor
     amazonCompile('com.amazon.android:amazon-maps-api:2.0')
+
+    //Apache http connection for images
```

Comment:
@barbeau thanks for the documentation. I added these libraries to be able to send images. In the city of bloomington open311 implementation (https://github.com/City-of-Bloomington/open311-android), these libraries were used to send images as an attachment with a report. In our application, this implementation worked for test servers, however it didn't worked for SeeClickFix (We already send them an email regarding this issue). 

I will investigate sending images with HttpURLConnection (It should be possible, since, we are sending just byte array). In the worst case, after SeeClickFix fixes their issues and when we test everything, I am planing the export this part as a library project as we discussed earlier.